ourjapanlife/findadoc-web
# feat: Adding cypress project id

Resolves #156


# What changed
- [x] Added cypress id to project
- [x] Random cleanup

# Testing instructions
Go to [cypress](https://cloud.cypress.io/organizations/a5eb7c28-2035-4549-a590-d11a93d4744a/projects) and see our project's test results uploaded
